.github/BOTMETA.yml

```diff
@@ -356,295 +357,324 @@ files:
     labels: cloud
   contrib/inventory/openstack.py:
     keywords:
-    - openstack dynamic inventory script
+      - openstack dynamic inventory script
     maintainers: $team_openstack
     labels:
-    - cloud
+      - cloud
   contrib/inventory/infoblox.py:
     keywords:
       - infoblox dynamic inventory script
     labels:
       - ipam
+      - nios
       - networking
     maintainers: $team_networking
   contrib/inventory/azure_rm.py:
     keywords:
-    - azure inventory
-    - azure rm inventory
-    - azure azure_rm dynamic inventory script
+      - azure inventory
+      - azure rm inventory
+      - azure azure_rm dynamic inventory script
     labels:
-    - cloud
-    - azure
+      - cloud
+      - azure
     maintainers: $team_azure
   contrib/inventory/ec2:
     keywords:
-    - aws dynamic inventory
-    - aws inventory
-    - ec2 inventory
-    - ec2 dynamic inventory
-    - ec2.py dynamic inventory script
-    - ec2.py inventory script
-    - ec2.py inventory
+      - aws dynamic inventory
+      - aws inventory
+      - ec2 inventory
+      - ec2 dynamic inventory
+      - ec2.py dynamic inventory script
+      - ec2.py inventory script
+      - ec2.py inventory
     labels:
-    - cloud
-    - aws
+      - cloud
+      - aws
     notify: ryansb s-hertel willthames
   contrib/inventory/vmware.py:
     keywords:
-    - vmware inventory
-    - vmware dynamic inventory script
+      - vmware inventory
+      - vmware dynamic inventory script
     labels:
-    - cloud
+      - cloud
     maintainers: $team_vmware
   contrib/inventory/vmware_inventory.py:
     keywords:
-    - vmware inventory
-    - vmware dynamic inventory script
+      - vmware inventory
+      - vmware dynamic inventory script
     labels:
-    - cloud
-    - vmware
+      - cloud
+      - vmware
     maintainers: $team_vmware
   lib/ansible/inventory:
     keywords:
-    - core inventory
-    - inventory
-    - inventory parsing
+      - core inventory
+      - inventory
+      - inventory parsing
+  $module_utils/azure_rm_common.py:
+    maintainers: $team_azure
+    labels:
+      - azure
+      - cloud
+  $module_utils/acme.py:
+    support: community
+    maintainers: resmo felixfontein
+  $module_utils/cloudstack.py:
+    support: community
+    maintainers: $team_cloudstack
+    labels: cloudstack
+  $module_utils/crypto.py:
+    support: community
+    maintainers: Spredzy
+  $module_utils/ipa.py:
+    support: community
+    maintainers: $team_ipa
+  $module_utils/k8s:
+    maintainers: chouseknecht maxamillion fabianvf flaper87 willthames
+    labels: clustering
+  $module_utils/keycloak.py:
+    support: community
+    maintainers: eikef
+  $module_utils/manageiq.py:
+    support: community
+    maintainers: $team_manageiq
   $module_utils/net_tools/nios/:
+    support: core
     labels:
       - networking
       - infoblox
     maintainers: $team_networking sganesh-infoblox
+  $module_utils/network:
```

Comment:
AFAIK, `support` key [isn't propagated to subpaths](https://github.com/ansible/ansibullbot/blob/f363f697da369fa123d701d74981fdc3d9a52c8e/ansibullbot/parsers/botmetadata.py#L113).